frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -71,16 +57,24 @@ const AdminJobsPage = () => {
                         <TestJobForm submitAction={submitTestJob} />
                     </Accordion.Body>
                 </Accordion.Item>
-                {
-                    jobLaunchers.map((jobLauncher, index) => (
-                        <Accordion.Item eventKey={index + 1}>
-                            <Accordion.Header>{jobLauncher.name}</Accordion.Header>
-                            <Accordion.Body>
-                                {jobLauncher.form}
-                            </Accordion.Body>
-                        </Accordion.Item>
-                    ))
-                }
+                <Accordion.Item eventKey="1">
+                   <Accordion.Header>Milk The Cows</Accordion.Header>
+                   <Accordion.Body>
+                       <JobComingSoon/>
+                   </Accordion.Body>
+               </Accordion.Item>
+               <Accordion.Item eventKey="2">
+                   <Accordion.Header>Instructor Report</Accordion.Header>
+                   <Accordion.Body>
+                       <InstructorReportJobForm submitAction={submitTestJob} />
+                   </Accordion.Body>
+               </Accordion.Item>
+               <Accordion.Item eventKey="3">
+                   <Accordion.Header>Update Cow Health</Accordion.Header>
+                   <Accordion.Body>
+                       <UpdateCowHealthJobForm submitAction={submitTestJob} />
+                   </Accordion.Body>
+               </Accordion.Item>
             </Accordion>
```

Comment:
Why are you undoing this good work?

Just change this:

```
 const jobLaunchers = [
        {
            name: "Update Cow Health",
            form: <JobComingSoon />
        },
        {
            name: "Milk The Cows",
            form: <MilkTheCowsJobForm/>
        },
        {
            name: "Instructor Report",
            form: <JobComingSoon />
        },
    ]
 ```

to this: 

```
const jobLaunchers = [
        {
            name: "Update Cow Health",
            form: <UpdateCowHealthJobForm submitAction={submitTestJob} />
        },
        {
            name: "Milk The Cows",
            form: <MilkTheCowsJobForm submitAction={submitMilkTheCowsJob} />
        },
        {
            name: "Instructor Report",
            form: <InstructorReportJobForm submitAction={submitInstructorReportJob} />
        },
    ]
```

And then you can leave the rest of the code alone.

Further: you could actually add:

```
{
            name: "Test Job",
            form: <TestJobForm submitAction={submitTestJob} />
        },
```

And then delete these lines.  Must less redundant, much cleaner dryer code.

```
              <Accordion.Item eventKey="0">
                    <Accordion.Header>Test Job</Accordion.Header>
                    <Accordion.Body>
                        <TestJobForm submitAction={submitTestJob} />
                    </Accordion.Body>
                </Accordion.Item>
```